routing_model/src/scenario_tests.rs

```diff
@@ -906,13 +904,17 @@ mod dst_tests {
     }
 }
 
+//////////////////
+/// Src
+//////////////////
+
 mod src_tests {
```

Comment:
I was thinking later we could perhaps split these `mod *_tests` to their own files

Reply:
Could be, but I'm not 100% sure, each test is really testing the full system (i.e making sure other parts are not interfering, or make sure other parts are doing their job (Replace elders, relocate again...).

Reply:
Yes. Thought they test if from different perspectives. Actually my main motivation is for me it's much easier and faster when I can jump to the relevant file rather than scrolling back and forth. But then again maybe my workflow sucks ;)

Reply:
My thinking was that when we add these header style comments to separate code groups, it often suggests that there is a natural split at play